### DISCUSSION TITLE
Feature freeze for ESMValCore v2.6

### DISCUSSION THREAD

**Discussion by sloosvel** [ESMValGroup/ESMValCore#1605] [General]
@ESMValGroup/esmvaltool-developmentteam This is a reminder that the feature freeze for version 2.6 of ESMValCore is scheduled for Tuesday 7th of June.

There are currently 19 open issues in the 2.6 milestone. 9 of these issues have an open PR linked to them that need to be reviewed:

- #1545 
- #1549 
- #1562 
- #1567 
- #1574
- #1281 
- #1589 
- #1599 
- #1603
- #1554 
- #1625

It would be great to strive to have them merged before next Monday since the work they contain has already been developed.

The milestone also has  PR that are currently marked as a draft:

- ~#1048~ 
- ~#1542~ 
- ~~#1337~~ 

To the authors, please make sure your pull request  is marked as ready for review if you want your work to be included in the upcoming release.

The following issues have been added to the 2.6 milestone but have no PR associated to them:

- ~#430~
- ~#1128~
- ~#744~  
- ~#857~ 
- ~#863~ 
- ~#1156~
- ~#1180~ 
- ~#1210~
- ~#1247~ 
- ~#1298~ 
- ~#1335~ 
- ~#1387~ 
- ~#1388~ 
- ~#1493~ 
- ~#1238~ 
- ~~#781~~
- ~~#1593~~ --> moved to v2.7

I would like to ask people involved in those issues to decide whether or not it's feasible to develop the work required to fix them for this upcoming release. If so, please open a PR with your work. If not, I will move them to the next milestone. 

Finally, if there are any issues or PR that are ready but have not been listed, please make sure you add them to the milestone.

Thanks for your collaboration!

**Comment by zklaus** [comment]
Thanks, @sloosvel, great overview!

**Comment by sloosvel** [comment]
By the way, is the 6th a holiday for you?

**Reply by zklaus** [reply]
In Sweden yes, it is the national day.

**Reply by sloosvel** [reply]
It's also a holiday in here, I was just wondering if it would make sense to move the code freeze to Tuesday 7th.

**Reply by schlunma** [reply]
It's also a public holiday in Germany, so it might make sense to move the deadline.

**Comment by sloosvel** [comment]
Hi,

The code freeze day has arrived and there are still 5 open pull requests that are missing an approval to get merged. It would be helpful if these could get tested and merged before the end of the day. Help regarding that is very much welcome, as I don't have the permission to approve all of them.

The draft PRs and issues will be removed from the milestone as it is unlikely they will be finished for today. Feel free to add them back to future milestones when your work is ready.

Thank you for the collaboration.
